src/views/v2/rsvp/ari/form.php

```diff
@@ -18,6 +18,9 @@
  * @version 5.0.0
  */
 
+// Ensure $going is not undefined in Ajax call.
+$going = isset( $going ) ? $going : null;
```

Comment:
We can probably avoid modifying the template and rather setting the `null` [here](https://github.com/the-events-calendar/event-tickets/blob/master/src/Tribe/Tickets_View.php#L1232).

Every time we update a template we'll have to put a `@since TBD Description of the changes`, and then set the version number of the template to `@version TBD`. So that can be adjusted before the release.

Reply:
@juanfra I have attempted setting the null value for the $going variable in the Tribe__Tickets__Tickets_View class, but unfortunately, this doesn't effect the chang adequately. I have used the get method for retrieving the status as used in other template forms like so - [here](https://github.com/the-events-calendar/event-tickets/blob/master/src/views/v2/rsvp/form/form.php#L19).